lib/main_common.pm

```diff
@@ -597,7 +597,14 @@ sub load_jeos_openstack_tests {
     } else {
         loadtest 'publiccloud/ssh_interactive_start', run_args => $args;
     }
-    #    loadtest "jeos/image_info";
+
+    if (get_var('CI_VERIFICATION')) {
```

Comment:
`CI_` sounds like Continuous Integration to me. Shall we use `CLOUD_INIT_VERIFICATION`?